src/libEth/Eth.cpp

```diff
@@ -45,7 +43,7 @@ Json::Value populateReceiptHelper(std::string const &txnhash, bool success,
   ret["cumulativeGasUsed"] = gasUsed.empty() ? "0x0" : gasUsed;
   ret["from"] = from;
   ret["gasUsed"] = gasUsed.empty() ? "0x0" : gasUsed;
-  ret["logs"] = Json::arrayValue;
+  ret["logs"] = logs;
```

Comment:
Have you checked that the logs are coming in the format expected by the ETH APIs?